sunscreen_runtime/src/lib.rs

```diff
@@ -20,20 +22,27 @@ pub use crate::metadata::*;
 pub use run::*;
 pub use runtime::*;
 pub use serialization::WithContext;
+pub use zkp::RuntimeZkpBackend as ZkpBackend;
 
 use seal_fhe::{Ciphertext as SealCiphertext, Plaintext as SealPlaintext};
 use serde::{Deserialize, Serialize};
 use sunscreen_zkp_backend::BigInt;
 
 #[derive(Debug, Clone, PartialEq, Hash, Serialize, Deserialize, Eq)]
+//#[serde(tag = "type")]
 /**
  * The underlying backend implementation of a plaintext (e.g. SEAL's [`Plaintext`](seal_fhe::Plaintext)).
  */
 pub enum InnerPlaintext {
     /**
      * This plaintext wraps a SEAL [`Plaintext`](seal_fhe::Plaintext).
      */
-    Seal(Vec<WithContext<SealPlaintext>>),
+    Seal {
+        /**
```

Comment:
In the future, start using
```rust
/// this is a doc
/// comment
```
style doc comments. JS doc tends to give subtle annoying issues and we've agreed to consolodate around ///.